customizations/ndv4_rocm.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -ex
+
+# Place NDv4 topo file under /opt/microsoft/ndv4
+mkdir -p /opt/microsoft/ndv4
+
+# Link the NDv4 topology file into /opt/microsoft/ndv4/
+ln -sf /opt/microsoft/ndv4-topo.xml /opt/microsoft/ndv4/topo.xml
+
+## Set NCCL configuration file for NDv4
+bash -c "cat > /etc/nccl.conf" <<'EOF'
+NCCL_IB_PCI_RELAXED_ORDERING=1
+NCCL_TOPO_FILE=/opt/microsoft/ndv4/topo.xml
+EOF
+
+error_code=$?
+if [ ${error_code} -ne 0 ]
+then
+    echo "NVIDIA Fabic Manager Inactive!"
+    exit ${error_code}
+fi
+
+# ## Setup NVME devices
```

Comment:
remove commented out lines?

Reply:
these are present in ndv4.sh as well, should we remove them there?

Reply:
removed in both files.